packages/common/src/stores/AppController.ts

```diff
@@ -57,10 +58,10 @@ export default class AppController {
     config = await this.configService.validateConfig(config);
     config = merge({}, defaultConfig, config);
 
-    const accessModel = this.configService.calculateAccessModel(config);
-    const { integrationType, clientId, isSandbox, offers } = this.configService.calculateIntegrationData(config);
+    // calculate and set the integration type. This must happen before the AccountController initialization!
+    const integrationType = this.calculateIntegrationType(config);
 
-    useConfigStore.setState({ config, accessModel, integrationType, clientId, isSandbox, offers, loaded: true });
+    useConfigStore.setState({ config, loaded: true, integrationType });
```

Comment:
Maybe it's a bit more clean and correct if we move this outside of this function, back to `initializeApp` on L73 perhaps.

Reply:
Yes, I wanted to ensure that the loaded and integration state were both set at the same time 🤔